requirements.txt

```diff
@@ -1,2 +1 @@
 Cython==0.24.1
-numpy==1.11.1
```

Comment:
I use `requirements.txt` when setting up a virtual environment, which I do every time I start a new Python project. I'd like to keep `requirement.txt` unaltered. It's separate from `setup.py` and `.travis.yml`. Basically, it lays out all of the project's dependencies, implicit *and* explicit.